Makefile

```diff
@@ -60,7 +60,7 @@ LLVM_LIBDIR = $(shell $(LLVM_CONFIG) --libdir | sed -e 's/\\/\//g' -e 's/\([a-zA
 # Apparently there is no llvm_config flag to get canonical paths to tools,
 # so we'll just construct one relative to --src-root and hope that is stable everywhere.
 LLVM_GIT_LLD_INCLUDE_DIR = $(shell $(LLVM_CONFIG) --src-root | sed -e 's/\\/\//g' -e 's/\([a-zA-Z]\):/\/\1/g')/../lld/include
-LLVM_SYSTEM_LIBS=$(shell ${LLVM_CONFIG} --system-libs --link-static | sed -e 's/[\/&]/\\&/g')
+LLVM_SYSTEM_LIBS=$(shell ${LLVM_CONFIG} --system-libs --link-static | sed -e 's/[\/&]/\\&/g' | sed 's/-llibxml2.tbd//')
```

Comment:
Huh, I'm surprised this worked.  Regardless, I think the right fix is `sed 's/-llibxml2.tbd/-lxml/'` though, since we probably want to preserve the fact that `llvm-config` claims libxml2 is a required system lib.

Reply:
This problem also occurs on macOS x86-64 for me.

Reply:
If you configure LLVM with `-D LLVM_ENABLE_LIBXML2=OFF` at build time then this isn't necessary.

Reply:
Would be nice for the makefile to work for people who didn't do that though. I'll take Shoaib's suggestion.